controllers/kepler_controller.go

```diff
@@ -161,56 +161,56 @@ func (r *KeplerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 		}
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
-	/*
-		//if inst set to delete, pv must also be removed when it is released
-		if inst.GetDeletionTimestamp() != nil {
-			if ctrlutil.ContainsFinalizer(inst, keplerFinalizer) {
-				//errorDeployment := r.removeDeployment(logger, inst, ctx)
-				//if errorDeployment != nil {
-				//	return ctrl.Result{}, errorDeployment
-				//}
-
-				errorPVC := r.removePVC(logger, inst, ctx)
-				if errorPVC != nil {
-					return ctrl.Result{}, errorPVC
-				}
-				errorPV := r.removePV(logger, ctx)
-				if errorPV != nil {
-					return ctrl.Result{}, errorPV
-				}
-				//	Include Additional Finalizers here
-				// Remove PV finalizer
-				ctrlutil.RemoveFinalizer(inst, keplerFinalizer)
-				updateError := r.Client.Update(ctx, inst)
-				if updateError != nil {
-					return ctrl.Result{}, updateError
-				}
-			}
-			return ctrl.Result{}, nil
-		}
 
-		// Add finalizer for this CR
-		if !ctrlutil.ContainsFinalizer(inst, keplerFinalizer) {
-			ctrlutil.AddFinalizer(inst, keplerFinalizer)
-			updateErr := r.Client.Update(ctx, inst)
-			if updateErr != nil {
-				return ctrl.Result{}, updateErr
-			}
-		}
-	*/
+	//if inst set to delete, pv must also be removed when it is released
```

Comment:
just remove them for clean?

Reply:
I will remove this. But the reason this exists is that if the CRD is removed, all resources will be terminated automatically except for the PV and PVC. These lines will include a finalizer to remove the PV and PVC. However, I think it would be smarter if we just left this for the user to decide to remove the PV and PVC. We should instead include a script that could automatically remove the PV and PVC that is triggered by the user and will only activate if the CR does not exist. Thoughts?

Reply:
Or we can include a field (RemovePVUponDeletion) in the CRD which will be a boolean and if set to true will add the finalizer. Or we can completely ignore it and the PV/PVC will remain even if CR is deleted and it is up to user to address it.

Reply:
>We should instead include a script that could automatically remove the PV and PVC that is triggered by the user and will only activate if the CR does not exist. Thoughts?
@KaiyiLiu1234 Maintaining script that is not part of the operator code/pkg doesn't seems right to me. 
@SamYuan1990 Could you elaborate on your thoughts as to why we should not have the finalizer to cleanup PVCs?

Reply:
Actually this is a very minor change since the code for including the finalizer is there already. I will make the change here

Reply:
Sorry I just found I am been mentioned in a reference...
@piparul , @KaiyiLiu1234 my original comments is that we should avoid use comments as `/**/` but we should remove pv/pvc once the instance been removed.